ci/do_ci.sh

```diff
@@ -8,22 +8,21 @@ set -e
 echo "building using ${NUM_CPUS} CPUs"
 
 function bazel_release_binary_build() {
-  cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building..."
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt @envoy//source/exe:envoy-static.stripped.stamped
+  cd "${ENVOY_BUILD_DIR}"
```

Comment:
Should we merge this now or do you want to look into this today / tomorrow? Even with `Modified`, it's still better than what's currently in tree.